src/orchestration/Orchestration.ts

```diff
@@ -387,6 +419,62 @@ export class Orchestration {
         this.eventCache.recordEvent(eventType, eventData);
     }
 
+    public loadEvaluations(request: EvidentlyRequest) {
```

Comment:
loadEvaluations is actually synchronous because it is not waiting on the API call. It just starts the call and keeps an internal reference to the promise